assets/Product.wxs

```diff
@@ -72,6 +72,16 @@
                   Execute="deferred"
                   Return="ignore"
                   Impersonate="no" />
+    <SetProperty Id="DisableTelemetry"
+                 Before="DisableTelemetry"
+                 Sequence="execute"
+                 Value="&quot;[$(var.ProductDirectoryName)]pwsh.exe&quot; -NoProfile -ExecutionPolicy Bypass -Command &quot;[\[]System.Environment[\]]::SetEnvironmentVariable('POWERSHELL_TELEMETRY_OPTOUT', '1',  [\[]System.EnvironmentVariableTarget[\]]::Machine)&quot;" />
+    <CustomAction Id="DisableTelemetry"
```

Comment:
Can we use an [Environment element](https://wixtoolset.org/documentation/manual/v3/xsd/wix/environment.html)

Reply:
Good idea, I'd have to look into it in more detail.

Reply:
Would the environment element be able to undo the change to env variable on uninstall?
Slightly off-topic: Because preview and stable use the same environment variable, I am wondering if there is a case where a user would want to turn telemetry off only for stable but allow for preview. WDYT? @kilasuit

Reply:
Its possible that could be a potential use case though that should I think be asked in another issue

Reply:
The other issues with doing it the way your are doing it, it would only be disabled in the shortcut.  This might give someone a false impression that it's disabled always.  I think it should be system wide or not at all.

Reply:
We would need a new feature in PowerShell for per channel disable.  @JamesWTruher  should be involved in that.